index.bs

```diff
@@ -558,8 +555,20 @@ returns a new {{Blob}} object with bytes ranging from the optional <var>start</v
 up to but not including the optional <var>end</var> parameter, and with a {{Blob/type}} attribute
 that is the value of the optional <var>contentType</var> parameter. It must act as follows:
 
-1. Return the result of <a>slice blob</a> given <a>this</a>, <var>start</var>, <var>end</var>,
-   and <var>contentType</var>.
+1. Let <var>sliceStart</var> be null.
```

Comment:
Nit: you could initialize all three to null here at once to make it slightly shorter.

Reply:
Good point!